### ISSUE TITLE
Missing parentheses in queries generated by the refinement options of the graphical querying

### ISSUE THREAD

**Issue by vogti** [polypheny/Polypheny-DB#180]
The query should contain parentheses to group conditions. Otherwise, the result is either not intuitive or wrong.

I suggest adding parentheses around all conditions caused by one option and additional parentheses around all options for one column. For the blow example, this would mean:
```sql
WHERE (public.emps.commission BETWEEN 758 AND 1000
OR (public.emps.commission = 400
OR public.emps.commission = 500))
AND (public.emps.name = 'Eric'
OR public.emps.name = 'Theodore')
``` 
It can be discussed whether it makes more sense to connect the outer parentheses (options for a column) by an `AND` or by an `OR`. Intuitively I would say one expects an `AND` here.

![grafik](https://user-images.githubusercontent.com/1682115/73351289-f0be1600-428e-11ea-90bf-19a3ef0c0b82.png)

**Comment by isabelge** [comment]
Fixed by commit [20707898b578a8b198d04783c9b0e63323ca5039](url) in [Polypheny-UI](url)

When possible I used IN () instad of OR so that the SQL statement doesn't get that long.

`WHERE (public.emps.commission BETWEEN 758 AND 1000
OR public.emps.commission IN (400,500)) 
AND public.emps.name IN ('Eric', 'Theodore')`

**Comment by vogti** [comment]
There is now a bug for queries containing a join (and corresponding join conditions) and at least two additional where conditions. 

![grafik](https://user-images.githubusercontent.com/1682115/73452616-811d5900-436a-11ea-94c3-f5c7d110609e.png)

In this example, the generated query looks like this:
```sql
SELECT public.emps.commission, public.depts.name, public.emps.salary
FROM public.emps, public.depts
WHERE public.emps.deptno = public.depts.deptno
OR pub(lic.emps.commission = 400
OR public.emps.salary = 10000)
```

To reproduce this, you have to add a foreign key constraint.

**Comment by vogti** [comment]
Fixed in [5f78d8da4872da37550852936fdb3af5cb77a549](https://github.com/polypheny/Polypheny-UI/commit/5f78d8d).
